src/applications/vaos/tests/appointment-list/components/ConfirmedAppointmentDetailsPage/index.unit.spec.jsx

```diff
@@ -51,21 +54,20 @@ describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
   it('should show confirmed appointments detail page', async () => {
     const url = '/va/21cdc6741c00ac67b6cbf6b972d084c1';
     const today = moment.utc();
-
-    const appointment = getVAAppointmentMock();
-    appointment.attributes = {
-      ...appointment.attributes,
-      clinicId: '308',
+    const data = {
+      id: '21cdc6741c00ac67b6cbf6b972d084c1',
+      currentStatus: 'FUTURE',
```

Comment:
You could use `status: 'booked',` here and it should set FUTURE automatically

Reply:
Done.